data/transactions/logic/eval.go

```diff
@@ -1895,9 +1895,29 @@ func opPushInt(cx *EvalContext) error {
 	return nil
 }
 
+func opPushInts(cx *EvalContext) error {
+	intc, nextpc, err := parseIntImmArgs(cx.program, cx.pc+1)
+	if err != nil {
+		return err
+	}
+	finalLen := len(cx.stack) + len(intc)
+	if cap(cx.stack) < finalLen {
+		// Let's grow all at once, plus a little slack.
+		newStack := make([]stackValue, len(cx.stack), finalLen+4)
+		copy(newStack, cx.stack)
+		cx.stack = newStack
+	}
```

Comment:
I think we're doing this in three places now.  Maybe we should have a function, `grow` or `ensure`?